index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Gowani Pump Supply</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Google Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+
+    <!-- Style Sheet for Sticky Bootstrap Footer -->
+    <link href="css/sticky-footer.css" rel="stylesheet">
+
+    <!-- Animate CSS -->
+    <link rel="stylesheet" href="css/animate.css">
+
+    <!-- Custom Style Sheets -->
+    <link rel="stylesheet" href="css/style.css">
+
+  </head>
+
+  <body>
+
+  <nav class="navbar navbar-toggleable-sm navbar-inverse bg-faded fixed-top">
+
+  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span>&#x2630;</span>
+  </button>
+
+  <a class="navbar-brand" href="#main" style="font-family: 'Open Sans', sans-serif;">Gowani Pump Supply</a>
```

Comment:
* `font-family` should be set on `body`
 * Avoid using `style=` and just style from your CSS file.